src/Tribe/Views/V2/Messages.php

```diff
@@ -109,8 +108,18 @@ public static function for_key( $key, ...$values ) {
 				'There were no results found for <strong>"%1$s"</strong> this month. Try searching next month.',
 				'the-events-calendar'
 			),
-			// translators: %1$s: events (plural), %2$s: the formatted date string, e.g. "February 22, 2020". %3$s html link to next day with available events.
+			// translators: %1$s: events (plural), %2$s: the formatted date string, e.g. "February 22, 2020".
 			'day_no_results_found'             => __(
+				'No %1$s scheduled for %2$s.',
+				'the-events-calendar'
+			),
+			// translators: the placeholder is an html link to the next month with available events.
+			'month_no_results_found_w_ff_link'           => __(
+				'There were no results found for this view. %1$s',
+				'the-events-calendar'
+			),
+			// translators: %1$s: events (plural), %2$s: the formatted date string, e.g. "February 22, 2020". %3$s html link to next day with available events.
+			'day_no_results_found_w_ff_link'             => __(
```

Comment:
:warning: Array double arrow not aligned correctly; expected 3 space(s) between "'day_no_results_found_w_ff_link'" and double arrow, but found 13.

<sub>hash: 9025823a2fdf7b92ee0915f935013b33</sub>